network.go

```diff
@@ -523,6 +524,35 @@ func createLink(netHandle *netlink.Handle, name string, expectedLink netlink.Lin
 	return getLinkByName(netHandle, name, expectedLink)
 }
 
+func createMqLink(netHandle *netlink.Handle, name string, expectedLink netlink.Link) (netlink.Link, []*os.File, error) {
```

Comment:
Instead of adding this function, don't you think we could add a flag for the multiqueue to `createLink()`:
```go
func createLink(netHandle *netlink.Handle, name string, expectedLink netlink.Link, mq bool)
```

Reply:
@sboeuf I should have removed the old method completely. It is no longer needed. We will be switching to fd based methods for all networking interfaces as far as possible. This will give us more control on how we setup the interfaces. i.e. QEMU will not be performing any networking setup internally.